__fixtures__/includes/withIncludes.conf

```diff
@@ -0,0 +1,28 @@
+# Follow an approach of:
+#
+# Service
+# Inputs
+# not valid include (file does not exist)
+
+@include nested/tail.conf
+# This will be ignored by the schema but considered by the AST
+@include nested/service.conf
+
+# @include alsoNested/in-functionC-log.conf
+
+# @include nonexistent/utopia.conf
+
```

Comment:
What about a `@include includes/withFailingIncludes.conf` here to test that nesting?

Reply:
https://github.com/calyptia/fluent-bit-config-parser/issues/19